README.md

```diff
@@ -29,15 +32,39 @@ gcloud beta functions deploy openhabGoogleAssistant --stage-bucket staging.<PROJ
 * This commands will deploy the function to Google Cloud and give you the endpoint address. Keep the address somewhere, you'll need it (something like `https://us-central1-<PROJECT ID>.cloudfunctions.net/openhabGoogleAssistant`).
 
 
+## Create OAuth Credentials on Google Cloud
+
+You'll need to create OAuth credentials to enable API access.
+* Visit the [Credentials Page](https://console.cloud.google.com/apis/credentials)
+  1. Select "Create Credentials" -> "OAuth client id"
+  1. Select Web Application and give it a name. I left the restrictions open.
+* Copy the client id and the client secret, you'll need these in the next step.
+
+
+
 ## Actions on Google
 
-Actions on Google is Googles platform for developers to extend Google Assistant. Here you need to develop your actions to engage users on Google Home, Pixel, and other surfaces where the Google Assistant is available
+Actions on Google is Google's platform for developers to extend Google Assistant. Here you need to develop your actions to engage users on Google Home, Pixel, and other surfaces where the Google Assistant is available.
 
-* Create and setup an Actions on Google project on the Actions Console using the Actions SDK described [here](https://developers.google.com/actions/sdk/create-a-project).
+* Create and setup an "Actions on Google" project on the [Actions Console using the Actions SDK](https://console.actions.google.com/).
+  1. Select your existing project
+  1. Select "Smart Home Actions". The fulfilment URL is the one saves from the `glcoud beta functions` you saved earlier.
+  1. Fill out all the App information. Feel free to use fake data and images, you're not actually going to submit this.
+  1. Move on to Account linking.
+    * Select Authorization Code
+    * Enter the client ID and client secret from the OAuth Credentials you created earlier
+    * Authorization URL should be something like: `https://openhab.myserver.com/oauth2/authorize`
```

Comment:
Please use generic pattern for URLs. Those URLs are misleading. THis project here can also be used with any other Cloud Backend beside openhab.

Reply:
I mention it should be like that. Not that it has to be that. What would you prefer it to say?